src/spectro_inlets_quantification/chip.py

```diff
@@ -99,8 +99,10 @@ def save(self, file_name: str, chip_dir: Optional[PATHLIKE] = None, **kwargs: An
         Args:
             file_name: Name of the .yml file. Should include the file suffix;
                 file_name.endswith(".yml")
-            chip_dir: path to directory to save chip in, defaults to :attr:`Config.chip_directory`
-                the spitze chips folder.
+            chip_dir: path to directory to save chip in. Defaults to
+                ``:attr:`config.aux_data_directory` / "chips"`` if the
+                `aux_data_directory` is set and if not defaults to
+                ``:attr:`config.data_directory` / "chips"``
             kwargs: (other) key word arguments are added to self_as_dict before saving.
         """
         file_name_with_suffix = Path(file_name).with_suffix(".yml")
```

Comment:
Should this also have the `Path(self.name + ".yml")` default?

Reply:
Wait, but shouldn't it? The name would typically be the serial number, presumably.

Reply:
It doesn't have to do with packaging, though. I've made an issue of it so we can merge this.
https://github.com/SpectroInlets/spectro_inlets_quantification/issues/6